### DISCUSSION TITLE
Does any of the converted Llama files allow for multiple gpu support?

### DISCUSSION THREAD

**Discussion by yopzey** [ggml-org/llama.cpp#3345] [General]
I have a rig with 8 a4000s and I am wondering if it even possible to run the 70b version on that. I have the 70b gguf model but it seems the model can't run across multiple gpu's. I new to this stuff no flame. Thanks for any help

**Comment by Green-Sky** [comment]
It should just work across multiple gpus, can you share your log? and did you build with cuda support?
